### ISSUE TITLE
Duda: ¿Cómo funciona el módulo con tarjeta de crédito?

### ISSUE THREAD

**Issue by diegosomar** [fedealvz/WHMCS-MercadoPago#23]
¿Hola todo bien? Tengo un producto de recurrencia mensual y me gustaría saber si cuando el cliente paga con tarjeta de crédito, todos los meses se le pagará automáticamente a la tarjeta de crédito utilizada en el primer pago o si todos los meses WHMCS generará una factura y el cliente tendrá que introducir manualmente los datos de la tarjeta?

¡Gracias!

**Comment by gonzad26** [comment]
No, el módulo actual solo conecta a Mercadopago. El cliente debe entrar a la invoice y pagarla.

Saludos.
